src/Microsoft.DotNet.Wpf/specs/AutomationUpdate.md

```diff
@@ -0,0 +1,503 @@
+```
+
+### **Parameters**
+**element** [DependencyObject]()
+The specified [DependencyObject]().
+
+**value**
+[Boolean]()
+**true** if the element should be identified as a dialog window; otherwise, **false**.
+
+
+## AutomationElement.IsDialogProperty Field
+Identifies the [IsDialog]() property
+
+```c#
+public static readonly System.Windows.Automation.AutomationProperty IsDialogProperty;
+```
+### **Field Value**
+[AutomationProperty]()
+
+### **Remarks**
+This identifier is for use by UI Automation client applications. UI Automation providers should use the equivalent field from [AutomationElementIdentifiers]().
+
+## AutomationElementIdentifiers.IsDialogProperty Field
+Identifies the [IsDialog]() property
+
+```c#
+public static readonly System.Windows.Automation.AutomationProperty IsDialogProperty;
+```
+### **Field Value**
+[AutomationProperty]()
+
+### **Remarks**
+This identifier is for use by UI Automation providers. UI Automation client applications should use the equivalent field from [AutomationElement]().
+
+
+## ContentTextAutomationPeer.RaiseActiveTextPositionChangedEvent(TextPointer, TextPointer) Method
+
+Raises an ActiveTextPositionChanged event.
+
+```c#
+public virtual void RaiseActiveTextPositionChangedEvent(TextPointer rangeStart, TextPointer rangeEnd);
+```
+
+### **Parameters**
+**rangeStart** [TextPointer]()
+The start of the range that changed.  
+The value must be a valid [TextPointer]() for the peer's owner, or `null` to denote the start of the owner's text content.
+
+**rangeEnd** [TextPointer]()
+The end of the range that changed.
+The value must be a valid [TextPointer]() for the peer's owner, or `null` to denote the end of the owner's text content.
+
+
+## TextAutomationPeer.RaiseActiveTextPositionChangedEvent(TextPointer, TextPointer) Method
+
+Raises an ActiveTextPositionChanged event.
+
+```c#
+public virtual void RaiseActiveTextPositionChangedEvent(TextPointer rangeStart, TextPointer rangeEnd);
+```
+
+### **Parameters**
+**rangeStart** [TextPointer]()
+The start of the range that changed.  
+The value must be a valid [TextPointer]() for the peer's owner, or `null` to denote the start of the owner's text content.
+
+**rangeEnd** [TextPointer]()
+The end of the range that changed.
+The value must be a valid [TextPointer]() for the peer's owner, or `null` to denote the end of the owner's text content.
+
+
+# Spec notes
+<!--
+This is an optional section.  It's often clearer to put non-public
+details in a separate section, rather than embed them in the public
+spec.  And it makes the doc team's job easier.
+-->
+All these APIs follow the patterns and naming conventions of previous UIA wrappers.
+
+All the properties and events are "passive", in the sense that they are provided for
+the use and convenience of applications, but WPF does not itself use them.
+With one exception:  WPF will automatically expose `true` as the value of `IsDialog` on a window that has been opened via `ShowDialog()`.
```

Comment:
Also `MessageBox`?

Reply:
WPF's MessageBox object is never in the automation tree.  It calls user32 MessageBox method - user32 is responsible for automation on the resulting hwnd.